content/guides/shimming.md

```diff
@@ -158,3 +159,86 @@ module.exports = {
 ## Node Built-Ins
 
 Node built-ins, like `process`, can be polyfilled right directly from your configuration file without the use of any special loaders or plugins. See the [node configuration page](/configuration/node) for more information and examples.
+
+
+## Loading polyfills on demand
+
+It's really common in web projects to include polyfills in the main bundle. This is not recommended since we are penalizing modern browsers users, since they will have to download a bigger file unnecessarely.
+
+The simplest way to mitigate this is by adding a separate entry point in your webpack config file including the polyfills your project needs.
+
+```javascript
+// webpack.config.js
+module.exports = {
+  entry: {
+    polyfills: [
+      'babel-polyfill',
+      'whatwg-fetch'
+    ],
+    main: './src/index.js'
+  }
+  // ... rest of your webpack config
+};
+```
+
+An alternative is to create a new entry file and manually import these packages.
+
+```javascript
+// src/polyfills.js
+import 'babel-polyfill';
+import 'whatwg-fetch';
+```
+
+```javascript
+// webpack.config.js
+module.exports = {
+  entry: {
+    polyfills: './src/polyfills.js',
+    main: './src/index.js'
+  }
+  // rest of your webpack config
+};
+```
+
+In your html file you need to conditionally load the `polyfills.js` file before your bundle. How you make this decision depends on the technologies and browsers you need to support.
+
+```html
+<script>
+  var modernBrowser = (
+    'fetch' in window &&
+    'assign' in Object
+  );
+
+  var scripts = [ '/main.js' ];
+
+  if (!modernBrowser) {
+    scripts.unshift('/polyfills.js');
+  }
+
+  scripts.map(function(src) {
+    var scriptElement = document.createElement('script');
+    scriptElement.async = false;
+    scriptElement.src = src;
+    document.head.appendChild(scriptElement);
+  });
+</script>
+```
+
+T> Any script added dynamically like in the example above will run as soon as it's parsed, but we need our polyfill to run before our bundle. This is why we are setting `async` to `false` for each script.
+
+
+### Smaller babel polyfill
+
+`babel-preset-env` uses [browserslist](https://github.com/ai/browserlist) to transpile only what is not supported in your browsers matrix. This preset comes with the `useBuiltIns` option _(false by default)_ which converts your global `babel-polyfill` import to a more granular feature by feature import pattern like:
```

Comment:
The CI build is failing currently because this link is a 404:

`https://github.com/ai/browserlist`

Was that repo moved?